gwcli.py

```diff
@@ -105,7 +105,7 @@ def main():
     root_node.refresh()
     if root_node.error:
         print("Unable to contact the local API endpoint "
-              "({})".format(settings.config.api_endpoint))
+              "({})".format(settings.config.local_api))
```

Comment:
Wait, shouldn't it be `self.local_api`?

Reply:
It's root_node.local_api. Fixed.

Reply:
[root@node02 ~]# gwcli  ls
Unable to contact the local API endpoint (http://localhost:5000/api)